json-from-csv/handler.py

```diff
@@ -32,6 +16,46 @@ def run(event, context):
     csvSet.writeEventData({ "data": csvSet.result_json })
     return event
 
+# retrieve configuration from parameter store
+def get_config():
+    client = boto3.client('ssm')
+    paginator = client.get_paginator('get_parameters_by_path')
+    path = '/all/stacks/mellon-manifest-pipeline/'
+    page_iterator = paginator.paginate(
+        Path = path,
+        Recursive=True,
+        WithDecryption=False,
+        MaxResults=4,)
+
+    response = []
+    for page in page_iterator:
+        response.extend(page['Parameters'])
+    
+    config = {}
+    for ps in response:
+        value = ps['Value']
+        # change height/width values from strings to numeric
+        if(ps['Name'].endswith('canvas-default-height') or
+            ps['Name'].endswith('canvas-default-width')):
+                value = int(ps['Value'])
+        # complete the partial value in parameter store with OS value
+        if(ps['Name'].endswith('image-server-base-url')):
+            value = ps['Value'].replace('STUB', os.environ['IMAGE_SERVER_URL'])
+        # complete the partial value in parameter store with OS value
+        if(ps['Name'].endswith('manifest-server-base-url')):
+            value = ps['Value'] + os.environ['MANIFEST_URL']
+        # change /all/stacks/mellon-manifest-pipeline/<key> to <key>
+        key = ps['Name'].replace(path,'')
+        # add the key/value pair
+        config[key] = value
+
+    config['process-bucket'] = os.environ['PROCESS_BUCKET']
+    config['image-server-bucket'] = os.environ['IMAGE_BUCKET']
+    config['manifest-server-bucket'] = os.environ['MANIFEST_BUCKET']
+    config['notify-on-finished']: os.environ['NOTIFY_EMAILS']
```

Comment:
This is maybe not an issue for this pull request but this needs to be read out of the csv.  Because it is intended to be the person to notify when the pipeline finishes for each manifest run.

Reply:
I've updated the PR so that the cloudformation creates default parameter store values.